.github/ISSUE_TEMPLATE/resourcescheduler.yml

```diff
@@ -0,0 +1,46 @@
+name: Request to schedule events to increase compute
+description: Schedule specific dates/times when your coursework requires higher computing resources
+labels: "support"
+title: Increase compute resources for Course XXX between specific date(s)/time(s)
+assignees:
+  - shaneknapp
+  - sean-morris
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please follow the sections in the template and provide all the required information for us to validate and allocate increased compute for your class.
+  - type: textarea
+    attributes:
+      label: Course Name
+      description: What is the name of the course?
+      value: |
+        <!--Include a link to appropriate webpage of the class if available. If not, please mention class name, instructor and a brief description.-->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Detailed Requirements
+      description: Rationale for requesting increased compute
+      value: |
+        - <!--Eg: Conducting Live Workshops/Exams/Flipped Classroom(s) or due to Deadline for Compute Intensive Assignment-->
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Session Details
+      description: Is this request for the current academic session?
+      options:
+      - "Yes"
+      - "No"
+      - Not Sure
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Request Deadline
```

Comment:
Request Start Date:
Request End Date:
Day(s) - This might be some sort of schedule(e.g. MWF) or just one day
Time - When do expect the group to begin logging into the hub?